src/components/editor/document-bar/document-info/document-info-time-line.tsx

```diff
@@ -19,16 +20,16 @@ export enum DocumentInfoLineWithTimeMode {
   EDITED
 }
 
-export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc }) => {
+export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc, isInline }) => {
   useTranslation()
 
   const i18nKey = mode === DocumentInfoLineWithTimeMode.CREATED ? 'editor.modal.documentInfo.created' : 'editor.modal.documentInfo.edited'
   const icon: IconName = mode === DocumentInfoLineWithTimeMode.CREATED ? 'plus' : 'pencil'
```

Comment:
I'm not happy anymore with the DocumentInfoLineWithTimeMode, because in react you should create generic components and then more specific ones. This means that we would need a "DocumentInfoTimeFromNowLine" (or something else), and then two specific components for "created" and "edited". Yes, I know that I created this component.. but I think it would be more correct. If you agree I would change this on my own.

Reply:
The difference is just the translation and the icon. I can understand that it doesn't follow the strict react-recommendations/rules but I fear that adding two more components that just contain their specific values are more complex to read/understand?

Anyway, if you want to change it, I'm not going to stop you.